src/test/commentCount.test.js

```diff
@@ -0,0 +1,20 @@
+import { calculateCommentLength } from '../modules/popUp.js';
```

Comment:
- [ ] [OPTIONAL] Part of the project requirement states that you should have tests for all edge cases. You could think at least about the scenario in which there are 0 items on the page (counter function should return 0)

Reply:
I have worked on this as well, thank you